### ISSUE TITLE
turf.union with empty polygon gives empty polygon

### ISSUE THREAD

**Issue by amagee** [Turfjs/turf#1259]
Using the turf version from here: https://npmcdn.com/@turf/turf@5.1.6/turf.min.js (the same as in the demos at http://turfjs.org/docs).

If I do something like

```
turf.union(turf.bboxPolygon([0, 0, 0, 0]), turf.bboxPolygon([0, 0, 1, 1]))
```

The result's `coordinates` is an empty array, whereas I would have expected it to be the same as ` turf.bboxPolygon([0, 0, 1, 1])`.

Interestingly, this works as expected:

```
turf.union(turf.bboxPolygon([999, 999, 999, 999]), turf.bboxPolygon([0, 0, 1, 1]))
```

**Comment by rowanwins** [comment]
Hi @amagee 

`turf/union` currently relies on the `jsts` library - we're currently looking at removing this dependency in favour of the `martinez` library as it is very large and can do weird things with some of the edges cases like yours.

Stay tuned

**Comment by rowanwins** [comment]
Resolved in v6.2.0

https://runkit.com/rowanwins/60050b242d8d3e001cebabc6
